### PR TITLE
Update virus-initiative-criteria.md

### PR DESCRIPTION
**Pull request by nbassett-MSFT**
Update to membership requirements, follow-up link for application, and high-level program summary.

### CODE REVIEW

File: windows/security/threat-protection/intelligence/virus-initiative-criteria.md [hunk 1]
```diff
@@ -18,34 +18,27 @@ ms.technology: mde
 
 # Microsoft Virus Initiative
 
-The Microsoft Virus Initiative (MVI) helps organizations to get their products working and integrated with Windows.
-
-MVI members receive access to Windows APIs and other technologies including IOAV, AMSI, and Cloud files. Members also get malware telemetry and samples and invitations to security-related events and conferences.
+The Microsoft Virus Initiative (MVI) helps organizations develop better-together security solutions that are performant, reliable, and aligned with Microsoft technology & strategy.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/virus-initiative-criteria.md:21]:
```suggestion
The Microsoft Virus Initiative (MVI) helps organizations develop better-together security solutions that are performant, reliable, and aligned with Microsoft technology and strategy.
```

File: windows/security/threat-protection/intelligence/virus-initiative-criteria.md [hunk 2]
```diff
@@ -18,34 +18,27 @@ ms.technology: mde
...
-You can request membership if you're a representative for an organization that develops and produces antimalware or antivirus technology. Your organization must meet the following requirements to qualify for the MVI program:
-
-1. Offer an antimalware or antivirus product that meets one of the following criteria:
-
-   * Your organization's own creation.
-   * Developed by using an SDK (engine and other components) from another MVI Partner company and your organization adds a custom UI and/or other functionality.
+You can request membership if you're a representative for an organization that develops and produces antimalware or antivirus technology. 
 
-2. Have your own malware research team unless you build a product based on an SDK.
+To qualify for the MVI program, your organization must meet all the following requirements.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/virus-initiative-criteria.md:27]:
```suggestion
To qualify for the MVI program, your organization must meet all the following requirements:
```

File: windows/security/threat-protection/intelligence/virus-initiative-criteria.md [hunk 3]
```diff
@@ -18,34 +18,27 @@ ms.technology: mde
...
 
-3. Be active and have a positive reputation in the antimalware industry.
+1)	Your security solution either replaces or compliments Microsoft Defender Antivirus.
 
-   * Activity can include participation in industry conferences or being reviewed in an industry standard report such as AV Comparatives, OPSWAT, or Gartner.
+2)	Your organization is responsible for both developing and distributing app updates to end-customers that address compatibility with Windows.
 
-4. Be willing to sign a non-disclosure agreement (NDA) with Microsoft.
+3)	Your organization must be active in the antimalware industry and have a positive reputation, as evidenced by participation in industry conferences or being reviewed in an industry standard report such as AV Comparatives, OPSWAT, or Gartner.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/virus-initiative-criteria.md:33]:
```suggestion
3)	Your organization must be active in the antimalware industry and have a positive reputation, as evidenced by participation in industry conferences or being reviewed in an industry-standard report such as AV-Comparatives, OPSWAT, or Gartner.
```

File: windows/security/threat-protection/intelligence/virus-initiative-criteria.md [hunk 4]
```diff
@@ -18,34 +18,27 @@ ms.technology: mde
...
+4)	Your organization must sign a non-disclosure agreement (NDA) with Microsoft.
 
-6. Be willing to adhere to program requirements for antimalware apps. These requirements define the behavior of antimalware apps necessary to ensure proper interaction with Windows.
+5)	Your organization must sign a program license agreement. Maintaining this license agreement requires that you adhere to all program requirements for antimalware apps. These requirements define the behavior of antimalware apps necessary to ensure proper interaction with Windows.
 
-7. Submit your app to Microsoft for periodic performance testing.
+6)	You must submit your app to Microsoft for periodic performance testing and feature review.
 
-8. Certified through independent testing by at least one industry standard organization.
+7)	Your solution must be certified through independent testing by at least one industry standard organization, and yearly certification must be maintained.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/virus-initiative-criteria.md:41]:
```suggestion
7)	Your solution must be certified through independent testing by at least one industry-standard organization, and yearly certification must be maintained.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nbassett-MSFT, kindly commit the minor suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nbassett-MSFT** [comment]
Sorry for the additional mail on this, but the site still shows the old content... is there anything I need to do?  I want to make sure I'm not blocking.

Thanks,
Nick
________________________________

**Comment by JohanFreelancer9** [comment]
@e0i, can you please help to move this PR forward? Many thanks.

**Pull request merged**